### PR TITLE
Bug Fix

### PR DESCRIPTION
**Pull request by Ranjith-K-07**
Date: 13-05-2022
Desc: Fixed missing Bug fixes and Implemented Eslint for the corresponding files.

### CODE REVIEW

File: src/components/Device/deviceConfiguration/subcomponent/DeviceConfigSetupModalComponent.js [hunk 1]
```diff
@@ -1,450 +1,455 @@
...
+            onClick={() => {
+              setOpen(false);
+              setErrorObject({});
+              loadData();
+            }}
+          >
+            Cancel
+          </Button>
+          <Button
+            // disabled={errorObject?.vendorName?.errorStatus || errorObject?.companyCode?.errorStatus || errorObject?.phoneNumber?.error
```
Comment (teamlead-rdl) [inline src/components/Device/deviceConfiguration/subcomponent/DeviceConfigSetupModalComponent.js:435]: Do we need this code as comments ? How exactly the disable work now for this button ?

File: src/components/maps/googleMapsComponent.jsx [hunk 2]
```diff
@@ -1,67 +1,66 @@
...
+  };
+  const [position, setPosition] = useState({
+    lat: props.longitude || 37.772,
+    lng: props.latitude || -122.214,
+  });
+  const [zoom, setZoom] = useState(15);
+
+  const onMarkerDragEnd = (event) => {
+    props.onMarkerDrop(event);
+    // setPosition({
```
Comment (teamlead-rdl) [inline src/components/maps/googleMapsComponent.jsx:19]: We can remove this commented code in case we had used this earlier for hard coding

**Review by teamlead-rdl — CHANGES_REQUESTED** [review]

File: src/components/Device/deviceConfiguration/subcomponent/DeviceConfigSetupModalComponent.js (hunk 1 above)
Comment (Ranjith-K-07) [reply src/components/Device/deviceConfiguration/subcomponent/DeviceConfigSetupModalComponent.js:435]: In this modal disabling submit button in not needed. We can remove these commented code.

Comment (teamlead-rdl) [reply src/components/Device/deviceConfiguration/subcomponent/DeviceConfigSetupModalComponent.js:435]: Sure, please remove the code if not needed. Other PR has been merged. You can carry on with this

**Review by teamlead-rdl — APPROVED** [review]

**Pull request merged**